Create rule S6265: Add JS as covered language (APPSEC-39) (#1084)
SonarSource/rspec: rules/S6265/javascript/rule.adoc

include::description.adoc[]

include::../ask-yourself.adoc[]

include::recommended.adoc[]

== Sensitive Code Example

> All users (ie: anyone in the world authenticated or not) have read and write permissions with the `PUBLIC_READ_WRITE` access control:

Reviewer: This change can also be applied to the other languages
```suggestion
All users, either authenticated or anonymous, have read and write permissions with the `PUBLIC_READ_WRITE` access control:
```
